src/types.ts

```diff
@@ -208,6 +209,14 @@ export interface SetExpiry extends AccountAction {
   expiryTime: Integer;
 }
 
+export interface Refund extends AccountAction {
+  receiverAccountOwner: address;
+  receiverAccountId: Integer;
+  refundMarketId: Integer;
+  otherMarketId: Integer;
```

Comment:
What is this? Isn't this just unused?

Reply:
Yes but we need to set it due to how AutoTrader is set up. And it cannot match refundMarketId. From a gas perspective, it should be the other market that was used in the trade.

Reply:
Right, but we should just set it in solo.js and not have clients pass it in

Reply:
or at least make it optional if you want to pass it in (if not passed autoset)